### DISCUSSION TITLE
path to self-signed certificates

### DISCUSSION THREAD

**Discussion by Ayuus** [blakeblackshear/frigate#12418] [Config Support] [answered]
### Describe the problem you are having

Using 0.14 beta3, I enabled TLS (strange: I hade to enable TLS explicitly, does not seem to enabled by default). In order to avoid the usual browser warning messages about invalid cedrtificates, I would like Frigate to use my self-signed certs.
How should I enter that in Frigate's config file?

### Version

0.14.0-beta3

### Frigate config file

```yaml
tls:
  enabled: true
```

### Relevant log output

```shell
N/A
```

### Operating system

Other Linux

### Install method

Docker Compose

### Object Detector

Coral

### Any other information that may be helpful

Could not find this in the docs, am I missing something?

**Comment by NickM-27** [comment, marked as answer]
TLS is enabled by default and that is fixed after beta3, the TLS docs have also been updated https://deploy-preview-11419--frigate-docs.netlify.app/configuration/tls

**Reply by Ayuus** [reply]
Exactly the docs I was looking for, thanks!

**Reply by Ayuus** [reply]
Works like a charm, thank you!

**Comment by GMagician** [comment]
Is this required also by mqtts? I used different volume name, set config but I can't find where to add config for ca certificate (self signed).

**Reply by GMagician** [reply]
Got it...in new configuration
